charts/fpl-case-service/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: fpl-case-service
 apiVersion: v1
 home: https://github.com/hmcts/fpl-ccd-configuration
-version: 1.11.1
+version: 1.11.2
```

Comment:
we might have to bump this as #1217 might be merged before this

Reply:
done